private/dependency_tree_parser.bzl

```diff
@@ -19,6 +19,8 @@ into target declarations (jvm_import) for the final @maven//:BUILD file.
 
 load("//:private/coursier_utilities.bzl", "SUPPORTED_PACKAGING_TYPES", "escape", "strip_packaging_and_classifier", "strip_packaging_and_classifier_and_version")
 
+JETIFY_INCLUDE_LIST_JETIFY_ALL = ["_PRIVATE_RULES_JVM_EXTERNAL_JETIFY_ALL_"]
+
```

Comment:
use `*`? 

this default value will show up in the generated `docs/api.md`.

Reply:
Great suggestion. I’ve struggled to suggest something more meaningful than an empty list / a magic value.

Reply:
ah, totally agree, will do